### ISSUE TITLE
Errors with certificates module

### ISSUE THREAD

**Issue by fridomac** [munkireport/munkireport-php#271]
If I use the certificates module, I get some errors in the preflight script (running on a 10.10.4 Server):

<code>
Munkireport: cert_check Error: Failed conversion of ``May 19 17:50:48 2017 GMT'' using format ``%b %e %T %Y %Z''
date: illegal time format
usage: date [-jnu] [-d dst] [-r seconds] [-t west] [-v[+|-]val[ymwdHMS]] ... 
            [-f fmt date | [[[mm]dd]HH]MM[[cc]yy][.ss]] [+format]
Failed conversion of ``Mar 5 08:44:30 2017 GMT'' using format ``%b %e %T %Y %Z''
date: illegal time format
usage: date [-jnu] [-d dst] [-r seconds] [-t west] [-v[+|-]val[ymwdHMS]] ... 
            [-f fmt date | [[[mm]dd]HH]MM[[cc]yy][.ss]] [+format]
Failed conversion of ``May 20 17:50:23 2017 GMT'' using format ``%b %e %T %Y %Z''
date: illegal time format
usage: date [-jnu] [-d dst] [-r seconds] [-t west] [-v[+|-]val[ymwdHMS]] ... 
            [-f fmt date | [[[mm]dd]HH]MM[[cc]yy][.ss]] [+format]</code>

I commented out the certificates module for now, just thought I´d report it!

Bye, Fridolin.

**Comment by bochoven** [comment]
I just copied the conversion code, the strange thing is that
```sh
date -j -f "%b %e %T %Y %Z" "May 19 17:50:48 2017 GMT" "+%s"
```
actually works.

Anyway you can figure out what goes wrong?

**Comment by fridomac** [comment]
If I run that command on the server that spit out the error message above, I get the same error. If I run it on my machine (10.10.4, too, System language set to german), the error persists. Tried setting the language to englisch -> same result. Seems to happen on all my machines, tried 10.8, 10.9, 10.10.
Will see what in that command makes the error happen and report back.

Bye, Frido.

**Comment by bochoven** [comment]
And what about

```sh
/bin/date -j -f "%b %e %T %Y %Z" "May 19 17:50:48 2017 GMT" "+%s"
```

**Comment by fridomac** [comment]
The same error. I cannot get it to run on any of my machines. Will try again.

**Comment by bochoven** [comment]
Fixed with 24f524c945237b622330abeddcd295b165410bd0
